docs/element-definitions.md

```diff
@@ -1,1419 +1,1623 @@
+}</pre>
+      </td>
+      <td>
+        <pre>[]</pre>
+        <pre>[42, "Hello world!"]</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#object-element">object</a> & <a href="#member-element">member</a></td>
+      <td>
+        <pre>{
+  "element": "object",
+  "content": [
+    {
+      "element": "member",
+      "content": {
+        "key": {
+          "element": "string",
+          "content": "foo"
+        },
+        "value": {
+          "element": "string"
+        }
+      }
+    },
+  ]
+}</pre>
+      </td>
+      <td>
+        <pre>{}</pre>
+        <pre>{
+  "foo": "Hey!"
+}</pre>
+        <pre>{
+  "bar": true
+}</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#enum-element">enum</a></td>
+      <td>
+        <pre>{
+  "element": "enum",
+  "attributes": {
+    "enumerations": {
+      "element": "array",
+      "content": [
+        {
+          "element": "string"
+        },
+        {
+          "element": "number"
+        }
+      ]
+    }
+  }
+}</pre>
+      </td>
+      <td>
+        <pre>-45.9</pre>
+        <pre>"Hello world!"</pre>
+        <pre>""</pre>
+        <pre>0</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#select-element">select</a> & <a href="#option-element">option</a></td>
+      <td>
+        <pre>{
+  "element": "select",
+  "content": [
+    {
+      "element": "option",
+      "content": {
+        "element": "string"
+      }
+    },
+    {
+      "element": "option",
+      "content": {
+        "element": "number"
+      }
+    }
+  ]
+}</pre>
+      </td>
+      <td>
+        <pre>-45.9</pre>
+        <pre>"Hello world!"</pre>
+        <pre>""</pre>
+        <pre>0</pre>
+      </td>
+    </tr>
+    <tr>
+      <td><a href="#extend-element">extend</a></td>
+      <td>
+        <pre>{
+  "element": "extend",
+  "content": [
+    {
+      "element": "object",
+      "content": [
+        {
+          "element": "member",
+          "content": {
+            "key": {
+              "element": "string",
+              "content": "foo"
+            },
+            "value": {
+              "element": "string"
+            }
+          }
+        },
+        {
+          "element": "member"
+          "content": {
+            "key": {
+              "element": "string",
+              "content": "bar"
+            },
+            "value": {
+              "element": "number"
+            }
+          }
+        }
+      ]
+    },
+    {
+      "element": "object",
+      "content": [
+        {
+          "element": "member"
+          "content": {
+            "key": {
+              "element": "string",
+              "content": "baz"
+            },
+            "value": {
+              "element": "boolean"
+            }
+          }
+        }
+      ]
+    }
+  ]
+}</pre>
+      </td>
+      <td>
+        <pre>{}</pre>
+        <pre>{
+  "foo": "Hey!"
+}</pre>
+        <pre>{
+  "bar": 42.3
+}</pre>
+        <pre>{
+  "foo": "Hey!",
+  "bar": 42.3,
```

Comment:
Let's change the values of `foo` and `bar` in the above 2 lines of this comment.